support/oidc-discovery-provider/README.md

```diff
@@ -30,16 +30,19 @@ The provider has the following command line flags:
 The configuration file is **required** by the provider. It contains
 [HCL](https://github.com/hashicorp/hcl) encoded configurables.
 
-| Key                | Type    | Required? | Description                              | Default |
-| ------------------ | --------| ---------| ----------------------------------------- | ------- |
-| `acme`             | section | required | Provides the ACME configuration. | |
-| `domain`           | string  | required | The domain the provider is being served from | |
-| `log_format`       | string  | optional | Format of the logs (either `"TEXT"` or `"JSON"`) | `""` |
-| `log_level`        | string  | required | Log level (one of `"error"`,`"warn"`,`"info"`,`"debug"`) | `"info"` |
-| `log_path`         | string  | optional | Path on disk to write the log | |
-| `log_requests`     | bool    | optional | If true, all HTTP requests are logged at the debug level | false |
-| `registration_api` | section | required\* | Provides Registration API details. Required unless the `workload_api` section is defined | |
-| `workload_api`     | section | required\* | Provides Workload API details. Required unless the `registration_api` section is defined | |
+| Key                  | Type    | Required? | Description                              | Default |
+| -------------------- | --------| ---------| ----------------------------------------- | ------- |
+| `acme`               | section | optional | Provides the ACME configuration. | |
```

Comment:
nit: maybe put `required*` on both acme and listen_socket_path and annotate your note below with the asterisk? If I was just scanning that list to get a feel for what I needed to configure, I might not notice the note below. The `*` might draw the eyes down to the qualifying note.

Reply:
Github flavored markdown doesn't support footnotes, but I've updated this with [1] style numbered-footnotes to indicate pairs where one or the other option must be set.